client/src/plugins/libparsec/definitions.d.ts

```diff
@@ -320,6 +320,29 @@ export type UserProfile =
   | UserProfileOutsider
   | UserProfileStandard
 
+// BackendAddrType
+export interface BackendAddrTypeInvitation {
+    tag: 'Invitation'
+}
+export interface BackendAddrTypeNormal {
```

Comment:
I don't like the word `Normal` here, it sounds weird. The other ones are not abnormal, they're just specializations. What about `BackendAddrTypeBase` instead?